x-pack/plugins/cases/public/components/use_push_to_service/index.test.tsx

```diff
@@ -182,27 +181,6 @@ describe('usePushToService', () => {
     expect(result.current.hasErrorMessages).toBe(true);
   });
 
-  it('Displays message when case is closed', async () => {
```

Comment:
nit: would be nice to verify 'updated incident...'  message verifying successful push after closed case in the unit test.

Reply:
Sure, will do :)